tests.sh

```diff
@@ -1,149 +1,148 @@
 #!/bin/bash
 
 # Extract version to build from the repo
-DEBVERSION=`grep Version debian/control | awk -F': ' '{print $2}'`
-DEBFILE="deb/thelounge_${DEBVERSION}_all.deb"
-NPMVERSION=`echo "${DEBVERSION}" | sed -E 's/-[0-9]+$//'`
+DEBVERSION=$(grep Version debian/control | awk -F': ' '{print $2}')
+DEBARCH=$(grep Architecture debian/control | awk -F': ' '{print $2}')
+DEBFILE="deb/thelounge_${DEBVERSION}_${DEBARCH}.deb"
+NPMVERSION=$(echo "${DEBVERSION}" | sed -E 's/-[0-9]+$//')
```

Comment:
Just curious, what differs between the backtick syntax and `$()`?

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2006